### ISSUE TITLE
TypeError: next is not a function

### ISSUE THREAD

**Issue by christianbundy** [ssbc/ssb-ws#19]
Looks like this is new in 6.1.0.

```
TypeError: next is not a function
    at Server.no_handler (/home/christianbundy/src/ssbc/patchwork/node_modules/ssb-ws/inject.js:20:7)
    at emitTwo (events.js:131:20)
    at Server.emit (events.js:214:7)
    at parserOnIncoming (_http_server.js:602:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:117:23)
```

**Comment by christianbundy** [comment]
This seems to be resolved, maybe from https://github.com/ssbc/ssb-ws/commit/b117bf0f499e4ef4e3b0c0f730ee5063a53dec5a?

**Comment by christianbundy** [comment]
Nope, looks like it's happening in 6.2.0 as well: https://github.com/ssbc/patchwork/pull/1020#issuecomment-489330071

cc: @dominictarr, do you know what might have broken this in ssb-ws@6.1.0?

**Comment by mixmix** [comment]
I'm getting this in Patchbay too. Must have accidentally upgraded, tried bumping to `6.2.0`, see it's still there.

I spent 20 minutes looking into this (reading code, tripping over unused and missing variables, refactoring). Didn't manage to get to a solution, but I have progressed this by connecting up some missing wiring that used to exist : https://github.com/ssbc/ssb-ws/pull/20

**Next steps?** : 
- review and merge my tidy/ fix
- dive into multiserver?
- investigate any changes introduced with major `multiblob-http` bump 

> ```
> -    "multiblob-http": "^0.4.2",
> +    "multiblob-http": "^1.0.0",
> ```
